docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -27,7 +27,7 @@ The following table maps specific authentication requirements to a supported mec
 
 |Username and password |xref:security-basic-authentication-concept.adoc[Basic], xref:security-authentication-mechanisms-concept.adoc#form-auth[Form]
 
-|Bearer access token |xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-jwt.adoc[JWT], xref:security-oauth2.adoc[OAuth2]
+|Bearer access token |xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer], xref:security-using-jwt-tutorial[JWT], xref:security-oauth2.adoc[OAuth2]
```

Comment:
Question @MichalMaler: I think the JWT file was one of the ones you mentioned earlier that was renamed. When I click on this link xref:security-using-jwt-tutorial[JWT], it doesn't open.